admin/client/EditorColorScaleSection.tsx

```diff
@@ -0,0 +1,430 @@
+import * as React from "react"
+import { action, IReactionDisposer, reaction, computed } from "mobx"
+import { observer } from "mobx-react"
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faPlus } from "@fortawesome/free-solid-svg-icons/faPlus"
+import { faMinus } from "@fortawesome/free-solid-svg-icons/faMinus"
+
+import { ColorScale } from "charts/ColorScale"
+import { ColorScaleBin, NumericBin, CategoricalBin } from "charts/ColorScaleBin"
+import { clone, noop } from "charts/Util"
+import { Color } from "charts/Color"
+
+import {
+    Section,
+    Toggle,
+    EditableList,
+    EditableListItem,
+    FieldsRow,
+    NumberField,
+    TextField,
+    ColorBox,
+    BindAutoFloat
+} from "./Forms"
+import { ColorSchemeOption, ColorSchemeDropdown } from "./ColorSchemeDropdown"
+
+@observer
+export class EditorColorScaleSection extends React.Component<{
+    scale: ColorScale
+}> {
+    render() {
+        return (
+            <React.Fragment>
+                <ColorsSection scale={this.props.scale} />
+                <ColorLegendSection scale={this.props.scale} />
+            </React.Fragment>
+        )
+    }
+}
+
+@observer
+class ColorLegendSection extends React.Component<{
+    scale: ColorScale
+}> {
+    @action.bound onEqualSizeBins(isEqual: boolean) {
+        this.props.scale.config.equalSizeBins = isEqual ? true : undefined
+    }
+
+    render() {
+        const { scale } = this.props
+        return (
+            <Section name="Legend">
+                <Toggle
+                    label="Disable visual scaling of legend bins"
```

Comment:
I understand this option for the map color legend, but does it make sense in the scatter color legend?

Reply:
It doesn't, but I'm hoping it will when we create better scatter legends. 

Right now I decided to get this live quicker and completely reuse the old scatter legend with its old, unchanged API, which has multiple issues (e.g. you have to write the labels yourself currently, otherwise you get unlabeled boxes by default). We can improve it if we feed the full `ColorScale#legendData` but I think we can do that later.

Also, it's not uncommon to have checkboxes that do nothing in the Grapher. Our internal tools have a pretty low bar for usability.

Reply:
Hid this for now anyway!